src/controllers/admin/evaluation-administration-controller.ts

```diff
@@ -298,7 +298,11 @@ export const generate = async (req: Request, res: Response) => {
         id: parseInt(id),
       },
     })
-
+    const evaluations = await prisma.evaluations.findMany({
+      where: {
+        evaluation_administration_id: parseInt(id),
```

Comment:
You can also add `for_evaluation: true` as a filter in `where`

Reply:
done :3